demisto_sdk/commands/content_graph/interface/neo4j/neo4j_graph.py

```diff
@@ -495,6 +496,13 @@ def find_core_packs_depend_on_non_core_packs(
             self._add_relationships_to_objects(session, results)
             return [self._id_to_obj[result] for result in results]
 
+    def find_mandatory_hidden_packs_dependencies(self, pack_ids):
```

Comment:
```suggestion
    def find_mandatory_hidden_packs_dependencies(self, pack_ids: List[str]):
```
please add an abstract method in graph.py, and a docstring